audiovisual/indico_audiovisual/plugin.py

```diff
@@ -79,6 +86,7 @@ class AVRequestsPlugin(IndicoPlugin):
     configurable = True
     settings_form = PluginSettingsForm
     default_settings = {'webcast_audiences': [],
+                        'notification_reply_email': None,
```

Comment:
I'd default to `''` to be consistent with the `webcast_url`

Reply:
Unrelated: `webcast_ping_url` should probably default to `None` or use the filter to convert empty to None in the form field...

Reply:
You mean it should default to `''`? It already defaults to `None`.

Reply:
yeah

Reply:
I can change that.